src/pages/home/report/ReportActionItem.tsx

```diff
@@ -392,6 +393,8 @@ function ReportActionItem({
 
     const attachmentContextValue = useMemo(() => ({reportID, type: CONST.ATTACHMENT_TYPE.REPORT}), [reportID]);
 
+    const mentionReportContextValue = useMemo(() => ({currentReportID: report?.reportID ?? ''}), [report?.reportID]);
```

Comment:
> fallback to -1 is better?

Same applies here.

Please merge main and all good

Reply:
@situchan updated, please move forward